packages/flutter/lib/src/material/input.dart

```diff
@@ -135,52 +144,30 @@ class _InputFieldState extends State<InputField> {
   }
 }
 
-/// A material design text input field.
-///
-/// Requires one of its ancestors to be a [Material] widget.
+/// Displays the visual elements of a material design text field. This
+/// widget's child is the input field widget itself.
```

Comment:
Only the first line is visible in the dartdoc overview pages, and I'm worried these two sentences make it sounds text-field specific (the next sentence explains that it's not, but that won't be visible).